modules/mysql/testcontainers/mysql/__init__.py

```diff
@@ -40,6 +40,25 @@ class MySqlContainer(DbContainer):
             ...     with engine.begin() as connection:
             ...         result = connection.execute(sqlalchemy.text("select version()"))
             ...         version, = result.fetchone()
+
+        The optional :code:`seed` param enables arbitrary SQL files to be loaded. This
+        is perfect for schema and sample data. The format is a tuple, made up of the
+        path to local data, and list of script files to load. Each script will be loaded
+        by exec in the container, using the MySQL root user, before yielding the
+        container. Any errors loading the scripts, will cause a ValueError.
+
+        .. doctest::
+
+            >>> import sqlalchemy
+            >>> from testcontainers.mysql import MySqlContainer
+            >>> seed_data = ("../../tests/", ["schema.sql", "data.sql"])
```

Comment:
Tradeoff: the docstring is ugly due to weird relative path to real sql files, but if I write what I mean:
```suggestion
            >>> seed_data = ("db/scripts/", ["schema.sql", "data.sql"])
```

Well then the doctests fail, which won't work either!